Policies/Network/allow-azurefirewallpolicy-user-defined-standard-ports-destiantionaddresses-within-network-rules/azuepolicy.rules.json

```diff
@@ -0,0 +1,72 @@
+{
+    "if": {
+        "allOf": [
+          {
+            "field": "type",
+            "equals": "Microsoft.Network/firewallPolicies/ruleCollectionGroups"
+          },
+          {
+            "count": {
+              "field": "Microsoft.Network/firewallPolicies/ruleCollectionGroups/ruleCollections[*]",
+              "where": {
+                "allOf": [
+                  {
+                    "field": "Microsoft.Network/firewallPolicies/ruleCollectionGroups/ruleCollections[*].FirewallPolicyFilterRuleCollection.rules[*].ruleType",
+                    "equals": "NetworkRule"
+                  },
+                  {
+                    "anyOf": [
+                      {
+                        "field": "Microsoft.Network/firewallPolicies/ruleCollectionGroups/ruleCollections[*].priority",
```

Comment:
This should be in a separate policy IMO. Having policies that enforce more than one thing can be confusing for users.
Also, is there a case where a low priority rule is allowed? (for example, if you want to do an emergency change) if so, this policy will block these cases as well